### PR TITLE
#65 Add some MySQL documentation

### PR DESCRIPTION
**Pull request by arey**
Remove some old txt files and put all the documentation into the readme.md
This PR required the changes https://github.com/spring-petclinic/spring-petclinic-microservices-config/commit/0361c87037425c4d4ee4614fe0df06640de479a2 into the Config Repository.
A spring `mysql` profile has been added.

On my laptop, MySQL use is working when I start microservices at command line or through my IDE. 
With Docker, the connection with the jdbc:mysql://localhost:3306/petclinic?useSSL=false fails. I suppose `localhost` is not known by docker. Maybe we have to add an optional mysql database into the docker-compose file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,21 +43,41 @@ Our issue tracker is available here: https://github.com/spring-petclinic/spring-
 
 ## Database configuration
 
-In its default configuration, Petclinic uses an in-memory database (HSQLDB) which
-gets populated at startup with data. A similar setup is provided for MySql in case a persistent database configuration is needed.
-Note that whenever the database type is changed, the data-access.properties file needs to be updated and the mysql-connector-java artifact from the pom.xml needs to be uncommented.
+In its default configuration, Petclinic uses an in-memory database (HSQLDB) which gets populated at startup with data.
+A similar setup is provided for MySql in case a persistent database configuration is needed.
+Dependency for Connector/J, the MySQL JDBC driver is already included in the `dpom.xml` files.
```
Comment (mszarlinski) [inline README.md]: typo: dpom -> pom

File: README.md [hunk 2]
```diff
@@ -43,21 +43,41 @@ Our issue tracker is available here: https://github.com/spring-petclinic/spring-
...
 
 ```
 docker run -e MYSQL_ROOT_PASSWORD=petclinic -e MYSQL_DATABASE=petclinic -p 3306:3306 mysql:5.7.8
 ```
+or download and install the MySQL database (e.g., MySQL Community Server 5.7 GA), which can be found here: https://dev.mysql.com/downloads/
+
+### Use the Spring 'mysql' profile
+
+To use a MySQL database, you have to start 3 microservices (`visits-service`, `customers-service` and `vets-services`)
+with the `mysql` Spring profile. Add the `--spring.profiles.active=mysql`` as programm argument.
```
Comment (mszarlinski) [inline README.md]: `` -> `

File: README.md (hunk 1 above)
Comment (arey) [reply README.md]: Fix

File: README.md (hunk 2 above)
Comment (arey) [reply README.md]: Fix. Thanks for the review.

**Pull request merged**